lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -355,63 +355,74 @@ private VerifiableCredentialBundle getVerifiableCredentials(
         var tacticalVcs = verifiableCredentialService.getVcs(userId);
 
         if (configService.enabled(EVCS_WRITE_ENABLED) || configService.enabled(EVCS_READ_ENABLED)) {
```

Comment:
As pointed earlier we don't need EVCS_WRITE_ENABLED check as only in case of EVCS_READ_ENABLED evcs will be source of truth for VCs.

Reply:
When only EVCS_WRITE_ENABLED then source of truth is tactical store....then no need to fetch VCs from EVCS.